docs/Governance.md

```diff
@@ -0,0 +1,223 @@
+as urgent security fixes.
+
+Regular maintainers do not routinely have access to Homebrew's hardware
+_(but can be granted access)_ and are not expected to do sysadmin work. Regular
+maintainers have access to CI for the purposes of build queue control &
+triggering jobs, and have the ability to manually request a rebuild via
+`@BrewTestBot test this please`.
+
+## Accountability
+
+Everyone is accountable for their words & actions, or lack thereof, in Homebrew,
+and nobody has unchecked power or influence. The below list covers various
+situations that may arise and how to handle them.
+
+### The Lead Maintainer Position & The Holder Of That Position
+
+At least once a year, provisionally September 14th, the PLC is mandated to
+affirm or withhold its consent for the lead maintainer role to exist in its
+current form as a distinct entity. The lead maintainer may vote on this issue.
+In the event of a tie regular maintainers are asked to vote on the issue and
+collectively act as the sole tie-breaking vote.
+
+On the same day the PLC is additionally mandated to vote on whether to affirm
+or withhold its consent for the current occupant of the lead maintainer role to
+remain in post. The lead maintainer _may not_ participate in this vote, and
+again in the event of a tie regular maintainers will be asked to provide a
+collective tiebreaker.
+
+Neither of the above bar any member of the PLC calling a vote on either issue
+at an earlier point in time, although doing so should be considered frowned
+upon in all but the most severe of circumstances. In both votes a simple
+majority suffices as a valid determination of the will of the PLC.
+
+If the lead maintainer position becomes vacant for any reason the PLC is
+mandated to fill the vacancy, unless a vote is held to absorb the position.
+Any maintainer or member of the PLC can nominate a new lead maintainer, but
+confirming a new lead maintainer requires every member of the PLC to vote
+and 2/3 of those votes to be in the affirmative.
+
+### Maintainer Pull Requests and Issues
+
+Maintainers may not close other maintainer's Pull Requests or Issues without
+the consent of the maintainer in question. The two exceptions of this rule are:
+
+_Inactivity_: If a Pull Request or Issue is inactive for 28 days, any maintainer
+is welcome to nudge the maintainer who opened the PR/Issue for an update on
+where things stand, and if one is not provided within two days the issue can be
+closed by any maintainer. If the maintainer who opened the PR/Issue wishes to
+return to it later they are welcome to reopen the PR/Issue.
+
+_Simple Changes_: If a maintainer submits, for example, a minor documentation
+tweak to `homebrew/brew` or a simple version bump to `homebrew/core` any
+maintainer is welcome to merge that in without waiting for the original
+maintainer to do so. If maintainers object to this they should add the
+`do not merge` label to the PR/Issue. That label may be ignored if the issue
```

Comment:
If the PR is “an urgent security fix or critical to the continued function of Homebrew's CI”, than it’s probably a bad idea to ignore `do not merge`. If the label is there, presumably the PR is not yet up to snuff and may break something else.

Reply:
My train of thought was something along the lines of:

1) Dave files a PR containing a critical security update for, say, `openssl`.
2) Dave tags this `do not merge` because they'd like to merge it in themselves, perhaps because they like to have all their merged commits GPG signed with their own key.
3) The CI build takes 5 hours to complete, in which time Dave has given up glaring at the build log & vanished offline. They don't return within an hour or so of the CI build finishing, and Sally decides to merge Dave's PR based on the CI result looking okay & the scale of changes being relatively minor.

I think Sally's actions would be justifiable there.